src/pages/citizenship.js

```diff
@@ -30,50 +30,56 @@ const ForeignSourceReporting = ({ data }) => {
         />
       </Helmet>
       <CreateStories stories={[oodLocalHeader]} />
-      <CenteredContainer>
-        <Heading level="h1" classes="su-mt-2 su-text-align-center">
-          Thank you for your recent gift
-        </Heading>
-        <p>
-          The federal government requires institutions of higher education, such
-          as Stanford, to report gifts from a donor who is not a U.S. citizen
-          when that donor makes gifts and other payments to Stanford with an
-          aggregate value of $250,000 or more in a calendar year. Stanford must
-          also report gifts and other payments received from a donor who has
-          citizenship in one of certain specific countries (the list may change
-          over time, and currently includes China (not including Hong Kong),
-          Russia, Iran, and North Korea), when that donor makes gifts and other
-          payments to Stanford having an aggregate value of $50,000 or more
-          during a reporting period that runs from July 1 to June 30 of the
-          following year. Your generous gift(s) meet one or both of these
-          reporting thresholds. Thank you for providing your information below.
-        </p>
-        <p>
-          You can find additional details about this policy in section V of
-          Stanford’s Gift Policy website:{' '}
-          <SbLink
-            linkType="url"
-            link={{
-              url: 'https://giving.stanford.edu/gift-policy',
+      <main id="main-content">
+        <CenteredContainer classes="su-max-w-800">
```

Comment:
I tossed a max width on the text to get closer to meeting 1.4.8. It's technically AAA, but it seemed like a good idea.